### DISCUSSION TITLE
http://skm.zoomquiet.top/data/20131021173330/index.html

### DISCUSSION THREAD

**Discussion by giscus** [zoom-quiet/comments#38] [Announcements]
# http://skm.zoomquiet.top/data/20131021173330/index.html

,BJD娃娃制作全过程

http://skm.zoomquiet.top/data/20131021173330/index.html

**Comment by ZoomQuiet** [comment]
|> 220726 贵方外链存在违规信息

http://skm.zoomquiet.top/data/20131021173330/060d39f6c8d9f2475e600817.jpg

七牛已对上述链接内容进行了屏蔽，请贵方收到本邮件后按照法律规范及时处置。同时，请贵方平台采取措施积极开展内容自检工作，防止此类情况再次发生。如多次出现违规情况，视情节严重程度，七牛云将依法依规调整处置措施，关停贵方违规域名或关停贵方违规账户。

为避免贵方正常业务经营受到影响，还请贵方予以重视，采取有效防范措施
